airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +64,349 @@ def get_value(args):
         print(value)
     except AirflowConfigException:
         pass
+
+
+class ConfigChange:
+    """Class representing the configuration changes in Airflow 3.0."""
+
+    def __init__(
+        self, section: str, option: str, suggestion: str = "", renamed_to: tuple[str, str] | None = None
+    ) -> None:
+        """
+        Initialize a RemovedConfig instance.
+
+        :param section: The section of the configuration.
+        :param option: The option within the section that is removed or deprecated.
+        :param suggestion: A suggestion for replacing or handling the removed configuration.
+        :param renamed_to: The new section and option if the configuration is renamed.
+        """
+        self.section = section
+        self.option = option
+        self.suggestion = suggestion
+        self.renamed_to = renamed_to
+
+    def get_message(self) -> str:
+        """Generate a message for this configuration change."""
+        lint_message = f"Removed deprecated `{self.option}` configuration parameter from `{self.section}` section. {self.suggestion}"
+
+        if self.renamed_to:
+            new_section, new_option = self.renamed_to
+            rename_message = f" Please use `{new_option}` from section `{new_section}` instead."
+            lint_message = lint_message + rename_message
+        return lint_message
+
+
+CONFIGS_CHANGES = [
```

Comment:
Wondering if we can get these dynamically instead?
We have newsfragments which can be the source. Just a thought to make this more manageable

Reply:
Sure, that makes sense. It was a thought I wanted to throw in. But I realised now that we have deprecated all that we should and so this serves well